src/contexts/currencyContext.ts

```diff
@@ -0,0 +1,21 @@
+import { createContext } from 'react'
+import {
+  V2_CURRENCY_ETH,
+  V2_CURRENCY_USD,
+  V2_CURRENCY_METADATA,
+} from 'utils/v2/currency'
+
+import { CurrencyMetadata } from 'constants/currency'
+
+export type CurrencyMetadataType = Record<number, CurrencyMetadata>
+
+export type CurrencyContextType = {
+  currencyMetadata: CurrencyMetadataType
+  currencies: Record<string, number>
```

Comment:
@johnnyd-eth can you pls create an issue to address this? Want to make sure we get to this eventually.

Reply:
Done